internal/shim-sgx/src/handler.rs

```diff
@@ -66,7 +66,7 @@ impl<'a> Handler<'a> {
             // prevent earlier writes from being moved beyond this point
             core::sync::atomic::compiler_fence(core::sync::atomic::Ordering::Release);
 
-            asm!("syscall");
+            asm!("cpuid");
```

Comment:
oops.. an oversight for a long time?

Reply:
No. It is a new code. It isn't even a bug. We just need to trigger an exception with an illegal opcode. We can actually use any opcode that is not valid in the case of SGX. But using CPUID here is slightly more self documenting.￼